hack/libsubid_tag.sh

```diff
@@ -7,9 +7,15 @@ mkdir -p "$tmpdir"
 trap 'rm -fr "$tmpdir"' EXIT
 cc -o "$tmpdir"/libsubid_tag -l subid -x c - > /dev/null 2> /dev/null << EOF
 #include <shadow/subid.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+const char *Prog = "test";
+FILE *shadow_logfd = NULL;
```

Comment:
nit: just curious if these two lines are getting used somewhere or was added for debugging purpose.

Reply:
they are needed by libsubid.